winui/BusyIndicator/Content.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Content with WinUI BusyIndicator control | Syncfusion
+description: Learn here all about showing the loading progress of the content with Syncfusion WinUI BusyIndicator Control.
```

Comment:
description: Learn all about showing the loading progress of the content with Syncfusion WinUI BusyIndicator Control here.

Reply:
Changed the content as mentioned above.